test/terra/noise/test_standard_errors.py

```diff
@@ -520,7 +520,7 @@ def test_pauli_error_2q_gate_from_string_1qonly(self):
         for i in range(actual.size):
             circ, prob = actual.error_term(i)
             expected_circ, expected_prob = expected.error_term(i)
-            self.assertEqual(circ, expected_circ)
+            self.assertEqual(qi.Operator(circ), qi.Operator(expected_circ))
```

Comment:
We can do:

```suggestion
            self.assertTrue(qi.Operator(circ).equiv(qi.Operator(expected_circ)))
```

if we want to ignore phase. But I don't think that matters here